editor.planx.uk/src/pages/FlowEditor/components/Settings/DesignSettings/index.tsx

```diff
@@ -0,0 +1,132 @@
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import { styled } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+import { Team } from "@opensystemslab/planx-core/types";
+import { FormikProps, getIn } from "formik";
+import { hasFeatureFlag } from "lib/featureFlags";
+import { useStore } from "pages/FlowEditor/lib/store";
+import React, { useEffect, useState } from "react";
+import EditorRow from "ui/editor/EditorRow";
+import { FeaturePlaceholder } from "ui/editor/FeaturePlaceholder";
+import InputGroup from "ui/editor/InputGroup";
+import InputLegend from "ui/editor/InputLegend";
+import ErrorWrapper from "ui/shared/ErrorWrapper";
+
+import { ButtonForm } from "./ButtonForm";
+import { FaviconForm } from "./FaviconFrom";
+import { TextLinkForm } from "./TextLinkForm";
+import { ThemeAndLogoForm } from "./ThemeAndLogoForm";
+
+export const DesignPreview = styled(Box)(({ theme }) => ({
+  border: `2px solid ${theme.palette.border.input}`,
+  padding: theme.spacing(2),
+  boxShadow: "4px 4px 0px rgba(150, 150, 150, 0.5)",
+}));
+
+export const EXAMPLE_COLOUR = "#007078";
+
+type SettingsFormProps = {
+  legend: string;
+  description: React.ReactElement;
+  input: React.ReactElement;
+  formik: FormikProps<any>;
+  preview?: React.ReactElement;
+};
+
+const useTeam = () => {
+  const [team, setTeam] = useState<Team>({} as Team);
+
+  useEffect(() => {
+    const fetchTeam = async () => {
+      try {
+        const fetchedTeam = await useStore.getState().fetchCurrentTeam();
```

Comment:
We need to get the team from the DB as the currently held team in the store does not include the theme. If we fetch the theme initially, it's them live throughout the Editor (e.g. the header changes colour when navigating from team to team).

This could be resolved and simplified, I'll either revist as another PR or open an issue.